packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2500,8 +2505,11 @@ export class StripeHelper extends StripeHelperBase {
       invoiceNumber,
       invoiceTotalInCents,
       invoiceTotalCurrency,
-      invoiceSubtotalInCents,
+      invoiceSubtotalInCents: showSubtotal ? invoiceSubtotalInCents : null,
       invoiceDiscountAmountInCents,
+      invoiceTaxAmountInCents: invoiceTaxAmountInCents
```

Comment:
This can be rewritten using [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator):
`invoiceTaxAmountInCents: invoiceTaxAmountInCents ?? null,`